libs/community/tests/unit_tests/document_loaders/test_recursive_url_loader.py

```diff
@@ -0,0 +1,91 @@
+import aiohttp
+import requests_mock
```

Comment:
I think `responses` is a more common library for mocking requests

Do we have both in the code base? https://github.com/getsentry/responses

Reply:
yea we already have requests-mock as test dep. no preference for which we use